tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,34,35},packaging
+envlist = py{27,34,35},docs,linters,packaging
 skip_missing_interpreters = True
 
```

Comment:
Please, add a general note why we are not using `deps = .[<extra-requirements>]` Tox option like [in Resolwe](https://github.com/genialis/resolwe/blob/8deae4379cb409b54d7114c74ac2ea191373d11f/tox.ini#L5-L6) and remove the notes from individual Tox environments.